src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1558,7 +1558,7 @@
     <comment>Text label for a button that, when invoked, sets the selected color scheme as the default scheme to use.</comment>
   </data>
   <data name="Globals_ConfirmCloseAllTabs.Header" xml:space="preserve">
-    <value>Confirm before closing all tabs</value>
+    <value>Warn on closing all tabs</value>

```

Comment:
My only concern is that it is different from how the feature works.
Here is my thought:

```suggestion
    <value>Warn when closing more than one tab</value>
```

Reply:
This makes it more concise
> 
> ```suggestion
>     <value>Warn when closing multiple tabs</value>
> ```